### PR TITLE
Confirmation prompt (dotnet-core-uninstall)

### PR DESCRIPTION
**Pull request by yuchong-pan**

### CODE REVIEW

File: src/dotnet-core-uninstall/LocalizableStrings.resx [hunk 1]
```diff
@@ -282,4 +267,31 @@
   <data name="ListX86OptionDescription" xml:space="preserve">
     <value>List x86 .NET Core SDKs or Runtimes.</value>
   </data>
+  <data name="DryRunOptionDescription" xml:space="preserve">
```
Comment (wli3) [inline src/dotnet-core-uninstall/LocalizableStrings.resx]: Remember to let Kathleen review the strings if you haven't

File: src/dotnet-core-uninstall/LocalizableStrings.resx [hunk 2]
```diff
@@ -282,4 +267,31 @@
...
+  <data name="YesOptionDescription" xml:space="preserve">
+    <value>Execute the command without requiring Y/n confirmation.</value>
+  </data>
+  <data name="DryRunOutputFormat" xml:space="preserve">
+    <value>*** DRY RUN OUTPUT
+Specified versions:
+{0}
+*** END DRY RUN OUTPUT
+
+To avoid breaking Visual Studio or other problems, read aka.ms/dotnet-core-uninstall.
```
Comment (wli3) [inline src/dotnet-core-uninstall/LocalizableStrings.resx]: Sounds a bit too casual to me

Comment (yuchong-pan) [reply src/dotnet-core-uninstall/LocalizableStrings.resx]: This has been run through @KathleenDollard.

File: src/dotnet-core-uninstall/LocalizableStrings.resx (hunk 1 above)
Comment (yuchong-pan) [reply src/dotnet-core-uninstall/LocalizableStrings.resx]: Yes, the strings have been run through @KathleenDollard.

**Pull request merged**